### DISCUSSION TITLE
Specify max & min value in CSV for HomeAssisatnt discovery

### DISCUSSION THREAD

**Discussion by GoNzCiD** [john30/ebusd#1100] [Q&A]
Hi all, is it possible to specify via csv the max and min value for a number entity and push it automatically for the HA's discovery?

thanks!

**Comment by dylan09** [comment]
Thats something I am also interested in. Someone a hint how to define the `min`,`max` and `step` values so that they could be used by the HA MQTT configuration?

**Reply by fwestenberg** [reply]
I’m interested in changing the `step` value as well. Everything else works fine with my Vaillant Arotherm Plus. 🥳
